package.json

```diff
@@ -72,7 +72,7 @@
     "jszip": "^3.7.0",
     "listr2": "^3.14.0",
     "minimist": "^1.2.5",
-    "node-fetch": "^3.1.0",
+    "node-fetch": "^2.6.6",
```

Comment:
For some reason nodejs 16 + ts_node + typescript don't allow to require node-fetch@3
https://stackoverflow.com/questions/69081410/error-err-require-esm-require-of-es-module-not-supported